JavaShared/src/main/java/org/openzen/zenscript/javashared/JavaNativeMethod.java

```diff
@@ -123,6 +127,7 @@ public enum Kind {
 		INTERFACE,
 		EXPANSION,
 		CONSTRUCTOR,
+		IMPLICIT_CONSTRUCTOR,
```

Comment:
💭 Question:  
Is a separate JavaNativeMethod.Kind appropriate for Implicit ctors?